packages/entities/entities-plugins/src/components/custom-plugins/PluginCustomGrid.vue

```diff
@@ -26,20 +26,28 @@
       class="custom-plugins-collapse"
       :data-testid="`${PluginGroup.CUSTOM_PLUGINS}-collapse`"
       :title="PluginGroup.CUSTOM_PLUGINS"
-      :trigger-label="triggerLabel"
+      :trigger-label="shouldCollapsedCustomPlugins ? t('plugins.select.view_more') : t('plugins.select.view_less')"
     >
       <!-- don't display a trigger if all plugins will already be visible -->
       <template
-        v-if="modifiedCustomPlugins.length <= pluginsPerRow"
+        v-if="!showCollapseTrigger"
         #trigger
       >
         &nbsp;
       </template>
+
+      <!-- not actually using KCollapse to hide/show content here, just using it for the trigger, title and container -->
```

Comment:
suggestion: do we really need all of these comments in the templates? These render in the DOM and are visible to customers.

Reply:
IMO we're not exposing anything we wouldn't want to expose in this comment

Reply:
Right but I’m not sure it’s necessary. No need to expose implementation details in the HTML when you can add comments in the script section if needed